src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -263,6 +263,11 @@ public enum JvmSettings {
     //CSL CITATION SETTINGS
     SCOPE_CSL(PREFIX, "csl"),
     CSL_COMMON_STYLES(SCOPE_CSL, "common-styles"),
+
+    // PersonOrOrgUtil SETTINGS
+    SCOPE_PERSONORORG(PREFIX, "person-or-org", "dataverse.personOrOrg"),
```

Comment:
Mostly curious - is the alias here needed?

Reply:
Hm, I am not sure but since I don't see any other scopes with aliases in that file, I guess not. It could be removed.